src/SDKs/RecoveryServices.SiteRecovery/Management.RecoveryServices.SiteRecovery/Generated/IOperations.cs

```diff
@@ -1,8 +1,4 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for
-// license information.
-//
-// Code generated by Microsoft (R) AutoRest Code Generator.
+// Code generated by Microsoft (R) AutoRest Code Generator 1.2.2.0
```

Comment:
@viverm you are not using the latest autorest. We no longer have this feature in the latest autorest.
If you use the latest, It will not touch all these files anymore.